dashboard/test/helpers/workshop_survey_results_helper_test.rb

```diff
@@ -143,4 +216,88 @@ class Pd::WorkshopSurveyResultsHelperTest < ActionView::TestCase
       summarize_workshop_surveys(workshops: [local_workshop, teachercon])
     end
   end
+
+  test 'daily survey get_question_for_forms gets workshop questions and substitutes question texts' do
+    assert_equal(@expected_questions, get_questions_for_forms(@workshop)
+    )
+  end
+
+  test 'generate workshop survey summary works as expected' do
+    common_survey_hash = {
+      form_id: CDO.jotform_forms['local']['day_0'],
```

Comment:
This should not be hardcoded. Instead call through [Pd::DailyWorkshopSurvey.get_form_id_for_day(0)](https://github.com/code-dot-org/code-dot-org/blob/d37236b428091be4aff59ca05b165960ef209755/dashboard/app/models/pd/workshop_daily_survey.rb#L68-L70), which will test another code path and also [raise](https://github.com/code-dot-org/code-dot-org/blob/d37236b428091be4aff59ca05b165960ef209755/dashboard/app/models/concerns/pd/jot_form_backed_form.rb#L117) a descriptive error if one of the config keys is missing.

Better yet, do we want to actually use the real ids in test? FWIW I have not added any configuration to chef for the test environment. We're not actually calling JotForm's API in the test, so why do we need real Ids? I'd rather just use a const fake form id here in the test that is self-contained.